### ISSUE TITLE
Why the reward model loss?

### ISSUE THREAD

**Issue by ghosthamlet** [openai/summarize-from-feedback#9]
Hi, thanks for the wonderful paper `Learning to summarize from human feedback`. I have a problem with the reward model loss, can you explain it more?

![Screenshot from 2021-02-09 12-39-28](https://user-images.githubusercontent.com/758325/107316643-0c7c0f00-6ad4-11eb-9326-8c269dbb8c2f.png)

I don't understand Why use this loss? when we minimize it, the two scalar output will get closer and closer, but shouldn't the two summaries have a very different score? As the one is better picked by humans when feedback, so the other is worse.

I have checked this codebase: https://github.com/openai/lm-human-preferences,
the mostlike loss is the PickBest, but it seems not fit the above loss formular:
```
class PickBest(LabelType):
    """Pick best response amongst N."""
    def __init__(self, num_responses):
        self.num_responses = num_responses

    def loss(self, reward_model, labels):
        logits = tf.stack([reward_model(labels['query'], labels[f'sample{i}'])
                         for i in range(self.num_responses)], axis=1)
        error = tf.reduce_mean(tf.nn.sparse_softmax_cross_entropy_with_logits(
            labels=labels['best'], logits=logits))
        return dict(loss=error, error=error)

    def question_schemas(self, *, query_length, response_length) -> Dict[str, Schema]:
        return dict(
            query=Schema(tf.int32, (query_length,)),
            **{f"sample{i}": Schema(tf.int32, (response_length,)) for i in range(self.num_responses)}
        )
```

**Comment by ghosthamlet** [comment]
@Rundong-Li i agree with you. It is a confusing typo problem in paper from the big OpenAI, can't believe it.
